projects/batfish/src/main/java/org/batfish/grammar/cisco/CiscoControlPlaneExtractor.java

```diff
@@ -1775,6 +1776,10 @@ public void enterS_mac_access_list_extended(S_mac_access_list_extendedContext ct
     _currentMacAccessList = list;
   }
 
+  @Override public void exitRo_rfc1583_compatibility(Ro_rfc1583_compatibilityContext ctx) {
+    _configuration.getCf().setRfc1583Compatible(ctx.NO() == null);
+  }
+
```

Comment:
My understanding here is that the method is only called if there's a line about RFC 1583 compatibility, and then ctx.NO() is null iff there's no `no` in that line.